src/Support/helpers.php

```diff
@@ -918,26 +918,31 @@ function title_case($value)
      * This function operates very similar to the PHP `realpath` function, except it will also work for missing files
      * and directories.
      *
-     * Returns canonical path if it can be resolved, otherwise `false`. Note that this function will always return
-     * UNIX-style paths.
+     * Returns canonical path if it can be resolved, otherwise `false`.
      *
      * @param  string  $path
      * @return string|bool
      */
     function resolve_path($path)
     {
         // Normalise directory separators
-        $path = str_replace('\\', '/', $path);
+        $path = str_replace(['\\', '/'], '/', $path);
```

Comment:
What's the logic behind this page?

Reply:
We might need to move this into a helper class that gets called from this function instead so that we can take advantage of methods to reduce / clarify some of the duplicated cleaning functionality

Reply:
@LukeTowers to make it easier to track and parse, I opted to change all directory separators while the path is being resolved to Unix `/` separators, regardless of OS. It's only when the final resolved path is generated and returned will the OS' directory separator be used.

Reply:
@LukeTowers I've converted the helper method to a Helper class (`October\Rain\Filesystem\PathResolver`) which might make it easier to track what is happening.